src/service/poc_lora.proto

```diff
@@ -0,0 +1,64 @@
+syntax = "proto3";
+
+package helium.poc_lora;
+
+message poc_lora_beacon_req_v1 {
+  bytes poc_key = 1;
+  bytes pub_key = 2;  
+  bytes entropy = 3;
+  bytes data = 4;
+  uint64 timestamp = 5;
+  sint32 signal = 6;
+  float snr = 7;
+  float frequency = 8;
+  int32 channel = 9;
+  string datarate = 10;
+  bytes addr_hash = 11;
+  int32 tx_power = 12;
+  bytes signature = 13;
+}
+
+message poc_lora_beacon_resp_v1 {
+}
+
+message poc_lora_witness_req_v1 {
+  bytes poc_key = 1;
+  bytes pub_key = 2;
+  bytes packet_hash = 3;
+  uint64 timestamp = 4;
+  sint32 signal = 5;
+  float snr = 6;
+  float frequency = 7;
+  int32 channel = 8;
+  string datarate = 9;
+  bytes signature = 10;
+}
+
+message poc_lora_witness_resp_v1 {
+}
+
+message poc_lora_valid {
+  bytes key = 1;
+  poc_lora_path_element path = 2;
+}
+
+message poc_lora_reject {
+    bytes key = 1;
+    bytes reject_reason = 2;
+    bytes reject_details = 3;    
+    oneof msg {
+        poc_lora_beacon_req_v1 beacon = 4;
+        poc_lora_witness_req_v1 witness = 5;
+  }
+}
+
+message poc_lora_path_element {
```

Comment:
why is this a "path element". Is there a better name for this?

Reply:
used old naming convention, will come up with something better suited to the new order

Reply:
I'm confused what this proto will be used for, is this the result of the collation?

Reply:
yes, this would be the result of the collation by the poc_verifier and would server as the rewardable POC